capstone/src/main/webapp/form-loader.js

```diff
@@ -0,0 +1,38 @@
+function callScriptFunction() {
+    var scriptId = "1Z_td2xr1Hq9loDzSdojcCS_3qFwKJR3apBuR2zmcyUVpdhqvfJWyMMYZ";
+  
+    // Call the Apps Script API run method
+    //   'scriptId' is the URL parameter that states what script to run
+    //   'resource' describes the run request body (with the function name
+    //              to execute)
+    gapi.client.script.scripts.run({
+    'scriptId': scriptId,
+    'resource': {
+        "function": "createForm",
+        "parameters": [],
+        "devMode": true
+    }
+    }).then(function(resp) {
+        var result = resp.result;
+        if (result.error && result.error.status) {
```

Comment:
You can use the questionmark operator to drill into paths which may be undefined.

var a = undefined;
a.b.c // Uncaught TypeError

a?.b?.c?.d // No error, just returns undefined

Reply:
Sounds like a cool concept. Will use